dockerV.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
+open http://localhost:3000
```

Comment:
This open command didn't work for me. If it needs to be installed, that should be done in the script, ideally with some logic to check if it's already been installed.
Also, this should be 4000 since 3000 on the Docker container maps to 4000 on the host.
```suggestion
open http://localhost:4000
```

Reply:
Fixed!